gramine_ratls_verify.go

```diff
@@ -210,13 +248,14 @@ func RATLSVerifyDer(certDER, mrenclave, mrsigner, isv_prod_id, isv_svn []byte) e
 
 // Verifies RA-TLS attestation x.509 PEM certificate
 func RATLSVerify(cert, mrenclave, mrsigner, isv_prod_id, isv_svn []byte) error {
+
 	if len(cert) == 0 {
-		return errors.New("empty PEM certificate")
```

Comment:
I prefere this more verbose error message, what is the reason for removing it?

Reply:
reasoning was to be consistent with the errors that we are returning from the library. Will add another error.